src/main/resources/elasticsearch/index/instance.json

```diff
@@ -18,9 +24,13 @@
         "tokenizer": "standard",
         "filter": [ "lowercase" ],
         "type": "custom"
+      },
+      "isbn_analyzer": {
+        "tokenizer": "standard",
+        "char_filter": [ "remove_hyphen_char_filter" ],
+        "filter": [ "lowercase" ]
```

Comment:
With the introduction of new `isbnSearchTermProcessor`, do you still need `isbn_analyzer`? It is not referenced anywhere.